src/Model/GroundWaterFlow/gwf3hfb8.f90

```diff
@@ -847,14 +901,23 @@ subroutine condsat_modify(this)
     real(DP) :: cond, condhfb
     real(DP) :: fawidth, faheight
     real(DP) :: topn, topm, botn, botm
+    real(DP) :: viscratio
 ! ------------------------------------------------------------------------------
     !
     do ihfb = 1, this%nhfb
+      ! -- initialize variable
+      viscratio = DONE
+      !
       ipos = this%idxloc(ihfb)
       cond = this%condsat(this%jas(ipos))
       this%csatsav(ihfb) = cond
       n = this%noden(ihfb)
       m = this%nodem(ihfb)
+      ! -- account for viscosity when active
+      if (this%ivsc /= 0) then
+!        call this%get_visc_ratio(n, m, hnew(n), hnew(m), viscratio)
```

Comment:
Is this line supposed to be commented out?

Reply:
The `condsat_modify` function didn't need to be modified on account of viscosity in the first place.  I commented this out for testing and then forgot to clean it out.  Restored to original in f23ddbc